apps/src/p5lab/redux/animationList.js

```diff
@@ -538,7 +558,8 @@ export function cloneAnimation(key) {
         saved: false
       })
     });
-    dispatch(selectAnimation(newAnimationKey));
+    const selector = type === 'background' ? selectBackground : selectAnimation;
```

Comment:
same nit: should we replace 'background' with `CurrentAnimationType.background`?

Reply:
there are a few more spots in this file where we might want to use `CurrentAnimationType` instead of a string